reana_client/cli/workflow.py

```diff
@@ -657,6 +666,15 @@ def workflow_logs(ctx, workflow, access_token, json_format):  # noqa: D301
                 click.echo(workflow_logs['workflow_logs'])
 
             first = True
+            returned_step_names = \
+                set(workflow_logs['job_logs'][item]['job_name']
+                    for item in workflow_logs['job_logs'].keys())
+            if len(returned_step_names) != len(steps):
+                for item in steps:
+                    if item not in returned_step_names:
+                        click.echo(
+                            click.style('Logs of step {} were not found.'
+                                        .format(item), fg='red'))
```

Comment:
I think this could be more clear if we directly use `set` operations:
```python
>> server_response = {'step_1': 'I am the step one', 'step_2': 'I am the step two', 'step_3': 'I am the step three'}
>> user_requested_steps = ['step_1', 'step_2', 'step_3']
>> missing_steps = set(user_requested_steps).difference(server_response.keys())
>> print(missing_steps)
{'step_3'}
```
What do you think? Then we could say `'The logs for {} could not be retrieved, check for spelling mistakes in the step names.'.format(', '.join(missing_steps))`